playbook/app/pb_kits/playbook/pb_passphrase/_passphrase.scss

```
@@ -0,0 +1,63 @@
+@import "../tokens/colors";
+@import "../tokens/spacing";
+@import "../tokens/screen_sizes";
+
+.pb_passphrase {
+  margin-bottom: $space_sm;
+
+  a.dark {
+    color: $white;
+  }
+
+  a {
+
+    &[class*=show-below-] {
+      display: none;
+    }
+    @each $breakpoint_name, $breakpoint in $breakpoints {
+      &.show-below-#{$breakpoint_name} {
+        @include break_at($breakpoint) {
+          display: inline;
+        }
+      }
+    }
+  }
+
+  .passphrase-label{
+    margin-right: 4px;
```

To avoid hard coded values can you try:
```suggestion
    margin-right: ($space_xs/2);
```
Same with the one below

---

That's perfect. It didn't occur to me that you can operate on things in scss. Making these updates